cfgov/cfgov/settings/local.py

```diff
@@ -40,7 +40,9 @@
 if os.environ.get('ENABLE_DEBUG_TOOLBAR'):
     INSTALLED_APPS += ('debug_toolbar',)
 
-    INTERNAL_IPS = ('127.0.0.1',)
+    INTERNAL_IPS = [
+        i for i in os.environ.get('INTERNAL_IPS', '127.0.0.1').split(' ')
```

Comment:
could this be written without the list comp and without the `split` arg, just: `os.environ.get('INTERNAL_IPS', '127.0.0.1').split()`?

Reply:
Any reason why we shouldn't just add `10.0.0.2` to this list (maybe with a comment informing what it is there for) and leave it at that? It wouldn't be needed in non-Docker environments but would be much simpler and would avoid needing to define this in `.env`.